### ISSUE TITLE
Pagination - text string update

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1777]
In our examples of the Pagination component, the element `button.pf-c-options-menu__toggle` should have `aria-label="Items per page"`.

This is for consistency with examples in React. https://github.com/patternfly/patternfly-react/issues/1882
